src/shell/components/shell-view.scss

```diff
@@ -11,6 +11,11 @@
     src: url('../../../assets/selawkl.woff2') format('woff2'),
          url('../../../assets/selawkl.woff') format('woff');
 }
+@font-face {
+    font-family: 'Selawik Semibold';
+    src: url('../../../assets/selawkl.woff2') format('woff2'),
+         url('../../../assets/selawkl.woff') format('woff');
+}
```

Comment:
Ack, you're right.  I'll work with @chetsangnil  to get the `sb` version of the font.